src/Console/Output/ProcessOutputInterface.php

```diff
@@ -19,5 +19,12 @@
  */
 interface ProcessOutputInterface
 {
+    public const OUTPUT_TYPE_NONE = 'none';
```

Comment:
I am up for having the constants/enums for those values.

Yet, I do not find it proper to have them declared in the interface.

`class FooOutput implements ProgressOutputInterface` - why would `FooOutput` be aware `OUTPUT_TYPE_DOTS` , at all? And then, when someone would add `FooOutput`, why they would need to touch the interface to add the `OUTPUT_TYPE_FOO` _to interface_ ?
and then imagine that it's non-internal class and one would like to register external , custom ProgressOutput class - and boom, they cannot adjust interface at source repo.

This should be outside of the interface and allow for extensions. Maybe mimic what we do for Fixers (and it's possibility to register custom fixers - not saying we need to introduce that possibility in this PR, but let's have a solution that will allow for that in future)